projects/canopy/src/lib/progress-indicator/progress-indicator.component.html

```diff
@@ -0,0 +1,12 @@
+<lg-progress-journey
+  [max]="max"
+  [value]="value"
+  [showAsPercentage]="showAsPercentage"
+  [stepsPrefix]="stepsPrefix"
+>
+  <ng-content />
+</lg-progress-journey>
+
+<lg-progress-bar *ngIf="showProgressBar" [max]="max" [value]="value" />
```

Comment:
Can we add a unit test for `showProgressBar` to ensure it's either rendered or not depending on the value?